zio-http-test/src/main/scala/zhttp/test/test.scala

```diff
@@ -0,0 +1,11 @@
+package zhttp
+
+import zhttp.http.{Http, HttpResult}
+import zio.ZIO
+
+package object test {
+  implicit class HttpWithTest[R, E, A, B](http: Http[R, E, A, B]) {
+    def apply(req: A): zio.UIO[HttpResult[R, E, B]] = ZIO.succeed(http.execute(req))
```

Comment:
```suggestion
    def apply(req: A): zio.ZIO[R, E, B] = http.execute(req).evaluate.asEffect
```

Something like this would be better.

Reply:
I think we were working on this at the same time, @tusharmath. https://github.com/dream11/zio-http/pull/308/commits/999f5decabb535c2fc1c0d57d4dff03230655ed9 made those exact changes :).